package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
     "eslint-config-nodesecurity": "^1.1.0",
+    "eslint-plugin-hapi": "^2.0.0",
```

Comment:
this shouldn't be needed, it's already a dependency of eslint-config-nodesecurity so it should already be installed